charmcraft/templates/init-simple/charmcraft.yaml.j2

```diff
@@ -0,0 +1,66 @@
+# This file configures Charmcraft.
```

Comment:
Is `main` the 4.0 branch? @dwilding In that case, isn't `init-simple` going away in 4.0?

Reply:
It is - the merge grabbed this as a non-conflict, but I've amended it away :-)